src/renderer/d3d.rs

```diff
@@ -435,7 +438,7 @@ impl SkiaRenderer for D3DSkiaRenderer {
                     size.width,
                     size.height,
                     DXGI_FORMAT_UNKNOWN,
-                    self.swap_chain_desc.Flags,
+                    DXGI_SWAP_CHAIN_FLAG::default(),
```

Comment:
I don't think this can be changed either, since it has to match the flags of the created swapchain, which is `Flags: DXGI_SWAP_CHAIN_FLAG_FRAME_LATENCY_WAITABLE_OBJECT.0 as u32`

Reply:
done